src/sgame/sg_bot_util.cpp

```diff
@@ -1198,15 +1198,18 @@ void BotGetIdealAimLocation( gentity_t *self, botTarget_t target, vec3_t aimLoca
 {
 	//get the position of the target
 	BotGetTargetPos( target, aimLocation );
+	bool target_buildable = BotGetTargetType( target ) == entityType_t::ET_BUILDABLE;
+	//would be better to just check if "not same team" but dunno how. Also, might try to kill "neutral"?
```

Comment:
`G_Team( self )` to get the bot's team.

Reply:
In the end, this code retrieves target's species, not really the team. I've updated comment to make that more clear.